src/main/windows/auth.ts

```diff
@@ -0,0 +1,67 @@
+import { BrowserWindow } from 'electron';
+
+import eventBus from '../event-bus';
+import { preloadPath, resolveHtmlPath } from '../util';
+import { setUpCommonWindowHandlers } from '.';
+import { getIsLoggedIn } from '../auth/handlers';
+
+let authWindow: BrowserWindow | null = null;
+export const getAuthWindow = () => {
+  return authWindow?.isDestroyed() ? null : authWindow;
+};
+
+export const createAuthWindow = async () => {
+  authWindow = new BrowserWindow({
+    width: 300,
+    height: 474,
+    show: false,
+    webPreferences: {
+      preload: preloadPath,
+      nodeIntegration: true,
+    },
+    movable: true,
```

Comment:
even if `movable` is set to true on the page need to have a section of it `-webkit-app-region: drag;` The `WindowTopBar` has it (the class draggable) but `Login` don't

Reply:
removed this